packages/richtext-lexical/src/packages/@lexical/markdown/MarkdownImport.ts

```diff
@@ -213,6 +213,15 @@ function $importBlocks(
 
   importTextTransformers(textNode, textFormatTransformersIndex, textMatchTransformers)
 
+  // Go through every text node in the element node and handle escape characters
+  for (const child of elementNode.getChildren()) {
+    if ($isTextNode(child)) {
+      const textContent = child.getTextContent()
+      const escapedText = textContent.replace(/\\([*_`~])/g, '$1')
+      child.setTextContent(escapedText)
+    }
+  }
+
```

Comment:
Shouldn't this be recursive? Or do you somehow know that the elementNode here only has TextNodes as children?

Reply:
Yep you're right

Reply:
Done! I moved it to `importTextTransformers` which will get recursively called for children or new sibling nodes created in the import process